src/sentry/static/sentry/app/components/commitLink.jsx

```diff
@@ -13,10 +13,16 @@ class CommitLink extends React.Component {
 
   getCommitUrl = () => {
     // TODO(jess): move this to plugins
-    if (this.props.repository.provider.id === 'github') {
+    if (
+      ['github', 'integrations:github'].indexOf(this.props.repository.provider.id) != -1
+    ) {
       return this.props.repository.url + '/commit/' + this.props.commitId;
     }
-    if (this.props.repository.provider.id === 'bitbucket') {
+    if (
+      ['bitbucket', 'integrations:bitbucket'].indexOf(
```

Comment:
I think you can instead use `Array.prototype.includes` ([MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)) to do the same thing without the `-1` check.